src/components/Announce/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import strings from 'lang';
+
+import blogpost from './blogpost';
+import styles from './styles.css';
+
+const BlogpostAnnounce = () => {
```

Comment:
I thought to open release notes in modal dialog onClick. But now I think it'll be better to make it similar.